tests/test_threaded_backend.py

```diff
@@ -131,6 +137,19 @@ def test_worker_error(self):
         self.assertTrue(self.backend.thread is None)
 
 
+class TestThreaded(object):
+    def test_calling_thread_waits_for_worker(self):
+        """
+        Start a process we are not the parent of and see if it waits for at
+        least the queue timeout (1) before exiting. If we were using a daemon
+        thread the process would exit immediately.
+        """
+        start = time.time()
+        subprocess.call("./tests/dummy_process.py")
+        delta = time.time() - start
+        assert delta >= 1
```

Comment:
I need to check the time difference as well. I'm testing if the process will wait for the thread to finish (i.e. 1 second). If the process runs without using a daemon thread it will exit within a few milliseconds.